modules/bling/installers/laptop.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# Tell build process to exit if there are any errors.
+set -oue pipefail
+
+wget "https://copr.fedorainfracloud.org/coprs/ublue-os/staging/repo/fedora-$(rpm -E %fedora)/ublue-os-staging-fedora-$(rpm -E %fedora).repo" \
+    -O "/etc/yum.repos.d/_copr_ublue-os_staging.repo"
+rpm-ostree remove power-profiles-daemon --install fprintd tlp tlp-rdw
```

Comment:
probably want to do `rpm-ostree override remove power-profiles-daemon --install fprintd tlp tlp-rdw`, there also could be issues with this too, as some fedora spins do not ship with `power-profiles-daemon` by default, so you may want to check if it's installed before removing it. Otherwise the build will fail

Reply:
Ohh yeah makes sense, maybe checking if it is installed with `rpm -qa` would be a nice idea